src/styles/card.scss

```
@@ -0,0 +1,119 @@
+$primary-color: #f3f3f3;
+$additional-color: #00acdc;
+$white-color: #fff;
+$dark-gray: #616070;
+$dark-blue: #060b35;
+
+p {
+  margin: 0;
+}
+
+.images-card {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid $primary-color;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition:
+    transform,
+    color,
+    300ms linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 0;
+    line-height: 18px;
+    color: $dark-blue;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    margin: 0;
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    color: $dark-gray;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    align-items: center;
+    line-height: 18px;
+
+    &-text {
+      color: $dark-gray;
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &-value {
+      color: $dark-blue;
+      font-size: 16px;
+      font-weight: 700;
+      text-align: right;
+      line-height: 18px;
+    }
+  }
+}
+
+.card__reviews {
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-top: 16px;
+  display: flex;
+}
```

```suggestion
&__reviews {
  justify-content: space-between;
  align-items: center;
  width: 100%;
  margin-top: 16px;
  display: flex;
}
```
nest the above into card